package.json

```diff
@@ -35,6 +35,7 @@
     "jquery": "2.1.4",
     "leaflet": "1.0.3",
     "moment": "^2.18.1",
+    "moment-timezone": "^0.5.13",
```

Comment:
I specified the exact same semver that in deep-insights. There we already have the ^. We can modify both repos. As you wish.